Views/Widget-Banner.liquid

```diff
@@ -4,7 +4,7 @@
 
 <header class="header" role="banner">
     <div class="header__content">
-        <a class="header__logo" href="/">
+        <a class="header__logo" href="{{ Request.PathBase }}/">
```

Comment:
I think this should be `Site.BaseUrl`, and is then configured within the admin dashboard `Configuration > Settings > General`, though it would be good why you've picked `Request.PathBase` instead

Reply:
I've used `Request.PathBase` because that will always be correct. Whereas, `Site.BaseUrl` is reliant on it being set within the admin area, which it often isn't.

Reply:
To be clear, `Request.PathBase` is purely catering for when a tenant is using a URL prefix instead of a hostname. When using a hostname, `Request.PathBase` will be empty, therefore the link will be `/`. When using `Site.BaseUrl`, if not filled in, the `href` would be an empty string, which would take the user to the page they are currently on (I think).

Reply:
I think you may be wrong with `Request.PathBase` when running a site on its own hostname.

I've found [an issue in dotnet core](https://github.com/dotnet/core/issues/1178#issuecomment-404883599) where somebody explains how some of the URI stuff works for `request`. The way I'm reading it if we're on a hostname site and we then browse to a `/blog` route, the `Request.PathBase` will then start returning `/blog`? 

Unless OC handles this differently it might be a good idea to test this with a hostname site, as I think this would be a better solution than relying on a setting.

Reply:
Don't think OC is doing anything fancy around `Request.PathBase`. If you navigate to any routes on a hostname tenant the `Request.PathBase` will be empty.

Included a screen recording of this in action:


![widgets-homepage-link](https://user-images.githubusercontent.com/651190/65493353-99580580-deaa-11e9-80af-d88478bebcfc.gif)